File: app/src/main/java/ru/eva/oriokslive/interfaces/http/MietAPI.java
package ru.eva.oriokslive.interfaces.http;

import retrofit2.Call;
import retrofit2.http.POST;
import retrofit2.http.Query;
import ru.eva.oriokslive.models.schedule.Schedulers;

public interface MietAPI {
    @POST("/schedule/data")
    Call<Schedulers> getScheduler(@Query("group") String group);
}

File: app/src/main/java/ru/eva/oriokslive/network/MietApi.kt
package ru.eva.oriokslive.network

import retrofit2.http.POST
import retrofit2.http.Query
import ru.eva.oriokslive.network.entity.schedule.Schedule

interface MietApi {
    @POST("/schedule/data")
    suspend fun getScheduler(@Query("group") group: String): Schedule?

    @POST("/schedule/groups")
    suspend fun getGroups(): Array<String>?
}
